examples/concertbot/config.yml

```diff
@@ -3,7 +3,7 @@ language: en
 pipeline: supervised_embeddings
 
 policies:
-  - name: KerasPolicy
+  - name: EmbeddingPolicy
     epochs: 200
     batch_size: 50
     max_training_samples: 300
```

Comment:
I don't think this parameter exists in embedding policy

Reply:
checked. It's part of `Policy` which in turn calls `featurize_for_training`